lib/report_generator.rb

```diff
@@ -10,8 +10,12 @@ def generate(query_id, query_params, recipients)
       query = DiscourseDataExplorer::Query.find(query_id)
       return [] unless query
 
+      creator = User.find(@creator_user_id)
```

Comment:
Do we want this to raise an error if the user cannot be found? If not, it is better to use `find_by` instead.

```suggestion
      creator = User.find_by(id: @creator_user_id)
```